designs/virtualization/4.6/Upload-image-prior-to-creating-VM/Upload Image prior to creating vm.md

```diff
@@ -0,0 +1,37 @@
+---
+parent: Virtualization
+version: 4.6
+---
+# Upload Image prior to creating a VM
+
+This PR deals with when a user wants to upload an image prior to creating a virtual machine.
+It needs to be accomplished via the storage/PVC area.
+It should ensure the user understands what the backing PVC does but does not remove the Disk terminology.
+
+The ‘Create PVC’ button will change into a dropdown, so users will be able to select ‘Upload data to Persistent Volume Claim’ from this dropdown. 
+
+![PVC list with drop down button](img/PVC-dropdown.png)
+
+They will get to a new upload screen with an inline info alert that says this PVC will be created using a Data Volume through Container Data importer (CDI). 
+User chooses an image to upload and fills in the PVC details: Name, size, storage class, access mode. Some of the fields can be filled out but editing will be optional as well.
+
+![browse to upload](img/Upload-data-to-pvc-1.png)
+
+After the user clicks ‘Upload’ they will get an inline notification with a spinner + text explanation about the PVC creation progress. We’ll use the spinner over the progress bar because we can't really offer progress and there aren't any steps to highlight.
+
+![Upload in progress notification](img/Upload-in-progress.png)
+
+Once the progress is successfully finished they will get a success indication with a request to keep this window open and can either choose to go to the PVC Details page or to the list view.
+
+![Success page](img/Upload-data-to-pvc-2.png)
```

Comment:
Does this conflict with the current "Create PVC" flow? Is there this confirmation page? Could we suggest it for all PVCs and contribute it?

Reply:
Currently, a 'new' PVC is just added to the PVCs list. We don't have any notification or confirmation page. So yes, I see your point in creating kind of a conflict here. We can either suggest this option for all PVCs and contribute it, OR omit it at all. @matthewcarleton WDYT?

Reply:
After today's Storage and CNV UX in OpenShift meeting, I guess there is no place for this success page anymore and the user will be transferred directly to the Details page. WDYT @lizsurette @matthewcarleton?

Reply:
Good point...seems like it should go directly to the PVC details.

Reply:
ya I agree - much simpler that way

Reply:
and I'll add a notification to keep the browser instance/tab open until uploading is done, so they won't miss an error state during the upload progress.